From bcba637666b545e0660df72cb8df76a518425641 Mon Sep 17 00:00:00 2001
From: "Satoshi N. M" <snmatsutake@yahoo.co.jp>
Date: Sat, 23 Dec 2017 17:28:51 -0800
Subject: [PATCH] Remove all_platforms, since only supporting one platform

---
 src/Makefile | 22 +---------------------
 1 file changed, 1 insertion(+), 21 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 52f3ed6..3804994 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -45,33 +45,13 @@ blackmagic.elf: include/version.h $(OBJ)
 	@echo "  OBJCOPY $@"
 	$(Q)$(OBJCOPY) -O ihex $^ $@
 
-.PHONY:	clean host_clean all_platforms FORCE
+.PHONY:	clean host_clean FORCE
 
 clean:	host_clean
 	$(Q)echo "  CLEAN"
 	-$(Q)$(RM) -f *.o *.d *~ blackmagic.elf $(HOSTFILES)
 	-$(Q)$(RM) -f platforms/*/*.o platforms/*/*.d mapfile include/version.h
 
-all_platforms:
-	$(Q)set -e ;\
-	mkdir -p artifacts/$(shell git describe --always) ;\
-	echo "<html><body><ul>" > artifacts/index.html ;\
-	for i in platforms/*/Makefile.inc ; do \
-		export DIRNAME=`dirname $$i` ;\
-		export PROBE_HOST=`basename $$DIRNAME` ;\
-		export CFLAGS=-Werror ;\
-		echo "Building for hardware platform: $$PROBE_HOST" ;\
-		$(MAKE) $(MAKEFLAGS) clean ;\
-		$(MAKE) $(MAKEFLAGS);\
-		if [ -f blackmagic.bin ]; then \
-			mv blackmagic.bin artifacts/blackmagic-$$PROBE_HOST.bin ;\
-			echo "<li><a href='blackmagic-$$PROBE_HOST.bin'>$$PROBE_HOST</a></li>"\
-				>> artifacts/index.html ;\
-		fi ;\
-	done ;\
-	echo "</ul></body></html>" >> artifacts/index.html ;\
-	cp artifacts/*.bin artifacts/$(shell git describe --always)
-
 command.c: include/version.h
 
 include/version.h: FORCE
