diff -ruN network-manager.orig/src/nm-device-wifi.c network-manager/src/nm-device-wifi.c
--- network-manager.orig/src/nm-device-wifi.c	2010-02-20 16:51:36.000000000 +0100
+++ network-manager/src/nm-device-wifi.c	2010-08-18 21:07:38.861447651 +0200
@@ -1714,15 +1714,13 @@
 	case NM_DEVICE_STATE_CONFIG:
 	case NM_DEVICE_STATE_NEED_AUTH:
 	case NM_DEVICE_STATE_IP_CONFIG:
-		/* Don't scan when unusable or activating */
+	case NM_DEVICE_STATE_ACTIVATED:
+		/* Don't scan when unusable, activating or already activated */
 		return FALSE;
 	case NM_DEVICE_STATE_DISCONNECTED:
 	case NM_DEVICE_STATE_FAILED:
 		/* Can always scan when disconnected */
 		return TRUE;
-	case NM_DEVICE_STATE_ACTIVATED:
-		/* Need to do further checks when activated */
-		break;
 	}
 
 	/* Don't scan if the supplicant is busy */
